test/modules/drawing-redraw.mjs

```
@@ -0,0 +1,74 @@
+import CRp from "../../src/extensions/renderer/canvas/drawing-redraw.mjs";
+import { expect } from "chai";
+
+describe('drawing-redraw', function(){
```

ESLint

[error] 'describe' is not defined.

(no-undef)